.github/workflows/ci-cd.yml

```diff
@@ -7,15 +7,15 @@ on:
   pull_request:
 
 env:
-  NODE_VERSION: 12.16
+  NODE_VERSION: 14
   PERCY_PARALLEL_NONCE: ${{ github.run_id }}-${{ github.run_number }}
   PERCY_PARALLEL_TOTAL: 1
 
 jobs:
   lint:
-    name: Lint files
+    name: Lint files and dependencies
     runs-on: ubuntu-latest
-    timeout-minutes: 10
+    timeout-minutes: 5
```

Comment:
I created a [script to analyze past workflow runs](https://github.com/ijlee2/inspect-workflow-runs) in order to recommend a good value for `timeout-minutes`.

(In general, if a workflow is to fail, we want it to _fail fast_ so that it doesn't cost us extra money. Currently, this is a concern only for private repos.)

I ran the script to find that `3` is a good value. Just in case, I set it to 5.